lib/data_pull.rb

```diff
@@ -222,4 +242,25 @@ def run(args:, include_missing:)
       )
     end
   end
+
+  class InspectorGeneralRequest
+    def run(args:, config:)
+      require 'data_requests/deployed'
+      ActiveRecord::Base.connection.execute('SET statement_timeout = 0')
```

Comment:
Is this safe because this is a one-off command invocation, thus not using a connection pool where this might affect other queries?

Reply:
Yeah it'll be run in its own process outside of any web requests